index.d.ts

```diff
@@ -15,6 +15,21 @@ declare module "replicate" {
     models?: Model[];
   }
 
+  export interface FileObject {
+    id: string;
+    name: string;
+    content_type: string;
+    size: number;
+    etag: string;
+    checksum: string;
+    metadata: Record<string, any>;
```

Comment:
From [TypeScript docs](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type):

> Constructs an object type whose property keys are Keys and whose property values are Type. This utility can be used to map the properties of a type to another type.

I've seen this used conventionally instead of something like this:

```typescript
interface Metadata {
  [key: string]: any;
}
```